2 - ListsLogicAndIterations/Code/Arrays.cs

```diff
@@ -6,11 +6,23 @@ public static class Arrays
 	{
 		public static int[] BuildIntArray(int fist, int second)
 		{
+            int[] Array = new int[] { fist, second};
+            return Array;
 			throw new NotImplementedException();
 		}
 
 		public static int GetArrayValue(int[] array, int position)
 		{
+            int result;
+            try
+            {
+                result = array[position];
+            }
+            catch (IndexOutOfRangeException)
+            {
+                result = 0;
+            }
+            return result;
```

Comment:
Though this does work, relying on exceptions for everything is expensive, slow and results in losing code flow control.

Array's have properties and methods to check its size and you can do some basic logic to see if a out of range would occur.